src/test/scala/example/HelloSpec.scala

```diff
@@ -1,4 +1,4 @@
-package example
+package rps
```

Comment:
Giusto cambiare il package, ma potresti rinominare anche la cartella a `rps`, come hai fatto dentro `src/main`. (In generale ha senso che le cartelle rispecchino la struttura dei package, anche se non è strettamente richiesto.)

Altra possibilità: eliminare completamente questo file e anche la dipendenza di ScalaTest (dal file Dependencies). Questo perché comunque quando aggiungeremo dei test lo faremo con MUnit, quindi tanto vale si può evitare di portarsi dietro quel codice ora.

Reply:
Sì, sono d'accordo, preferisco tenere la repo pulita da file inutili (o che sappiamo andremo a eliminare).